spring-boot-starter/spring-boot-starter-discovery/src/main/java/io/servicecomb/springboot/starter/discovery/CseDiscoveryClientConfiguration.java

```diff
@@ -17,33 +17,17 @@
 package io.servicecomb.springboot.starter.discovery;
 
 import org.springframework.boot.autoconfigure.AutoConfigureBefore;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
 import org.springframework.cloud.client.discovery.DiscoveryClient;
 import org.springframework.cloud.client.discovery.noop.NoopDiscoveryClientAutoConfiguration;
-import org.springframework.cloud.netflix.zuul.filters.ZuulProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Primary;
-
-import io.servicecomb.core.provider.consumer.ConsumerProviderManager;
+import org.springframework.core.annotation.Order;
 
 @AutoConfigureBefore(NoopDiscoveryClientAutoConfiguration.class)
 @Configuration
 public class CseDiscoveryClientConfiguration {
-
-  @Bean
-  public CseDiscoveryProperties cseDiscoveryProperties() {
-    return new CseDiscoveryProperties();
-  }
-
-  @Bean
-  @ConditionalOnBean(ZuulProperties.class)
-  public CseRoutesProperties cseRoutesProperties(ConsumerProviderManager manager) {
-    return new CseRoutesProperties(manager);
-  }
-
   @Bean
-  @Primary
+  @Order(5000)
```

Comment:
Is there any other client need to be found?

Reply:
Yes. Spring cloud provide a composite DiscoveryClient as Primary, we can't use this as primary and this will cause corruption.